docs/guide/tutorials/create-indexes.txt

```diff
@@ -1,3 +1,241 @@
 ==============
 Create Indexes
 ==============
+
+To create an index on a field or fields, pass an index specification
+document to the ``createIndex()`` method:
+
+.. code-block:: js
+
+
+      { <field1>: <type1>, <field2>: <type2> ... }
+
+Create an Ascending Index
+-------------------------
+
+For an ascending index type, specify ``1`` for ``<type>``.
+
+The following example creates an ascending index key for the
+``dateOfBirth`` field:
+
+.. code-block:: js
+
+   async function createAscendingIndex(db) {
+     // Get the users collection
+     const collection = db.collection('users');
+
+     // Create the index
+     const result = await collection.createIndex({ dateOfBirth : 1 });
+     console.log(result);
+     return result;
+   };
+
+Create a Descending Index
+-------------------------
+
+For an descending index type, specify ``-1`` for ``<type>``.
+
+The following example specifies a descending index key on the
+``lastName`` field:
+
+.. code-block:: js
+
+   async function createDescendingIndex(db) {
+     // Get the documents collection
+     const collection = db.collection('users');
+
+     // Create the index
+     const result = await collection.createIndex({ lastName : -1 });
+     console.log(result);
+     return result;
+   };
+
+Create a Compound Index
+-----------------------
+
+To specify a compound index, use the ``compoundIndex`` method.
+
+The following example specifies a compound index key composed of the
+``lastName`` field sorted in descending order, followed by the
+``dateOfBirth`` field sorted in ascending order:
+
+.. code-block:: js
+
+   async function createCompoundIndex(db) {
+     // Get the documents collection
+     const collection = db.collection('users');
+
+     // Create the index
+     const result = await collection.createIndex({ lastName : -1, dateOfBirth : 1 });
+     console.log(result);
+     return result;
+   };
+
+Create a Text Index
+-------------------
+
+MongoDB also provides
+:manual:`text </core/index-text/>` indexes to
+support text search of string content. Text indexes can include any
+field whose value is a string or an array of string elements.
+
+This example specifies a text index key for the ``comments`` field:
+
+.. code-block:: js
+
+   async function createTextIndex(db) {
+     // Get the documents collection
+     const collection = db.collection('users');
+
+     // Create the index
+     const result = await collection.createIndex({ comments : "text" });
+     console.log(result);
+     return result;
+   );
```

Comment:
Typo on line 94, it's a ")" instead of a closing bracket "}"

The full code should read:
```
async function createTextIndex(db) {
  // Get the documents collection
  const collection = db.collection('users');
  // Create the index
  const result = await collection.createIndex({ comments : "text" });
  console.log(result);
  return result;
}
```